pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -313,6 +318,7 @@ func (p *PriorityQueue) Add(pod *v1.Pod) error {
 	if err := p.podBackoffQ.Delete(pInfo); err == nil {
 		klog.ErrorS(nil, "Error: pod is already in the podBackoff queue", "pod", klog.KObj(pod))
 	}
+	klog.V(5).InfoS("Pod moved between internal queues", "pod", klog.KObj(pod), "event", PodAdd, "to", activeQName)
```

Comment:
how about:

```suggestion
	klog.V(5).InfoS("Pod moved to an internal queue", "pod", klog.KObj(pod), "event", PodAdd, "queue", activeQName)
```

Reply:
Changed to `Pod moved to an internal scheduling queue`.

Reply:
changed `to` to `queue`.